ibm/mas_devops/playbooks/cp4d/db2wh-backup-restore.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: localhost
+  any_errors_fatal: true
+  vars:
```

Comment:
All these should end up living in the role's default/main.yml ... see earlier examples provided

Reply:
Fixed.